README.md

```
@@ -449,20 +449,20 @@ Contributions are welcome! Please follow these guidelines:
 
-## 🔗 Links
+## Links
 
 - **Starknet**: https://starknet.io
 - **CMTAT**: https://www.cmtat.org
 - **OpenZeppelin Cairo**: https://github.com/OpenZeppelin/cairo-contracts
 
 ---
 
-**Built for compliant securities on Starknet 🚀**
+**Built for compliant securities on Starknet**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)